### PR TITLE
Add kickstart events and detail post

### PR DESCRIPTION
**Pull request by RealOrangeOne**
A part of https://github.com/srobo/tasks/issues/382

## To Do

- Confirm details of Cambridge kickstart
- Confirm the location of the Southampton kickstart

### CODE REVIEW

File: _events/sr2020/southampton-kickstart.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| Time  | Info |
+|-------|------|
+| 10:00 | Introductory presentation, where the game of SR2020 will be announced. |
+| 11:00 | Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30 | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+| 17:00 | Finish. |
+
+[teams-contact]: mailto:teams@studentrobotics.org
+[soton-food-map]: https://drive.google.com/open?id=1vDqr-yQh-OO7MzVa_OiuKD5xUrEsLprr
+[soton-directions]: {{ site.baseurl }}/resources/sr2019/southampton-kickstart-directions.pdf
```
Comment (PeterJCLaw) [inline _events/sr2020/southampton-kickstart.md]: This likely needs updating for this year; the PDF has "SR2019" baked in.

File: _events/sr2020/london-kickstart.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Schedule
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Introductory presentation, where the game of SR2020 will be announced. |
+| 11:00 | Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30 | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+| 17:00 | Finish. |
+
+[venue-map]: https://goo.gl/UzDEPv
```
Comment (PeterJCLaw) [inline _events/sr2020/london-kickstart.md]: Fixes SR2019 -> SR2020.
```suggestion
[venue-map]: https://drive.google.com/open?id=1O4S1mchOGnaCV2XxF97pE3hn0LEHr8zn
```

File: _posts/2019-09-16-sr2020-kickstart-details.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams. This year the
+kickstart event will be on the 26th October 2019.
+
+The venues for Student Robotics 2020 kickstart are:
+
+- The [University of Southampton][southampton-kickstart]
+- [Thread][thread]'s offices in [Aldgate, London][london-kickstart]
+- Cambridge (**TODO**)
+
```
Comment (PeterJCLaw) [inline _posts/2019-09-16-sr2020-kickstart-details.md]: How does the potential northern venue fit in here?

File: _events/sr2020/london-kickstart.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
+
+<!-- TODO: find out whether we can provide parking
+If you intend to travel by road, please [let us know][teams-contact] so we can
+arrange for parking.
+ -->
```
Comment (PeterJCLaw) [inline _events/sr2020/london-kickstart.md]: In theory I think we can offer parking, though this wasn't an issue last year. I don't think that anyone would really want to drive anyway. I suggest dropping these lines and then sorting it out if someone asks about parking.

File: _events/sr2020/southampton-kickstart.md (hunk 1 above)
Comment (RealOrangeOne) [reply _events/sr2020/southampton-kickstart.md]: Quite likely. It's in the description to be updated once we actually know where kickstart will be.

File: _posts/2019-09-25-sr2020-kickstart-details.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+The venues for Student Robotics 2020 kickstart are:
+
+- The [University of Southampton][southampton-kickstart]
+- [Thread][thread]'s offices in [Aldgate, London][london-kickstart]
+- [Cambridge][cambridge-kickstart] (Details TBC)
+
+If you would like a chance to [compete][compete] in Student Robotics 2020,
+please fill in the [entry form][entry-form] with the required information.
+We're expecting to run with a similar number of teams to SR2019, so places
+will go fast. Places close October 1st.
```
Comment (Adimote) [inline _posts/2019-09-25-sr2020-kickstart-details.md]: [nit]

```suggestion
will go fast. Applications close October 1st.
```

File: _events/sr2020/southampton-kickstart.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+title: Southampton Kickstart
+date: 2019-10-26 10:00:00
+layout: event
+type: kickstart
+location: Southampton University
```
Comment (Adimote) [inline _events/sr2020/southampton-kickstart.md]: [nit]

```suggestion
location: University of Southampton
```

**Pull request merged**